### PR TITLE
Get albums list

### PR DESCRIPTION
**Pull request by shernanm**
## Summary

* The endpoint is implemented that allows you to list the purchased albums.
* Their respective tests are carried out.
* Mocks are made for the response of the external service.

## Trello Card

https://trello.com/c/Bh9Ao9Yw

### CODE REVIEW

File: app/controllers/album.js [hunk 1]
```diff
@@ -54,3 +54,12 @@ exports.buyAlbums = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getAlbumsList = (req, res, next) => {
+  logger.info(`getAlbumsList method start, request methods: ${req.method}, endpoint: ${req.path}`);
+  return findAllAlbums(req.params.user_id)
```
Comment (raparicio6) [inline app/controllers/album.js]: we always use camel case

File: app/controllers/album.js [hunk 2]
```diff
@@ -54,3 +54,12 @@ exports.buyAlbums = (req, res, next) => {
...
+    .then(albums => {
```
Comment (raparicio6) [inline app/controllers/album.js]: remove { }

File: test/album.test.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
 
-const { signUp, signIn } = require('./utils/users');
+const app = require('../app');
+const { signUp, signIn, createAdmin } = require('./utils/users');
 const { albumMock } = require('./mocks/albums');
 
-const app = require('../app');
 const controller = request(app);
 
+albumMock();
```
Comment (raparicio6) [inline test/album.test.js]: you should put this somewhere, for example in a before each

**Review by raparicio6 — CHANGES_REQUESTED** [review]

File: app/controllers/album.js [hunk 4]
```diff
@@ -53,3 +53,10 @@ exports.buyAlbums = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getAlbumsList = (req, res, next) => {
+  logger.info(`getAlbumsList method start, request methods: ${req.method}, endpoint: ${req.path}`);
+  return findAllAlbums(req.params.userId)
```
Comment (molinajulian) [inline app/controllers/album.js]: Use findAllBy . Is more scalable

File: package.json [hunk 5]
```diff
@@ -64,6 +64,7 @@
     "jsonwebtoken-promisified": "^1.0.3",
     "jwt-simple": "^0.5.1",
     "morgan": "^1.9.1",
+    "nock": "^10.0.6",
```
Comment (molinajulian) [inline package.json]: Nock is dev-dependency. Also, delete this because you are use jest

**Review by molinajulian — CHANGES_REQUESTED** [review]

**Review by molinajulian — APPROVED** [review]

**Pull request merged**